src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <div class="box">
+        <img
+          class="img"
+          src="images/Search.svg"
+          alt="attr-req-value"
+        >
+        <input 
+          class="input input--big"
```

Comment:
```suggestion
          class="input"
```
where do you have this class?